# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/D3D11ShareHandleImage.cpp
# Commit: 75b04de7e99c
# Full Hash: 75b04de7e99cab7bed5cc8b0f2f6d52ddf174bf3
# Author: Jean-Yves Avenard <jyavenard@mozilla.com>
# Date: 2019-05-25 09:39:13
# Regressor Bug: 1550422
# File Overlap Count: 1
# Description:
#   Bug 1550422 - P15. Move Skip and Once gfxPrefs to StaticPrefs. r=jrmuizel
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D31259
# ==============================================================================

diff -r 91c3acdb2454 -r 75b04de7e99c gfx/layers/D3D11ShareHandleImage.cpp
--- a/gfx/layers/D3D11ShareHandleImage.cpp	Fri May 24 11:32:20 2019 +0000
+++ b/gfx/layers/D3D11ShareHandleImage.cpp	Fri May 24 11:32:54 2019 +0000
@@ -10,8 +10,8 @@
 #include "WMF.h"
 #include "d3d11.h"
 #include "gfxImageSurface.h"
-#include "gfxPrefs.h"
 #include "gfxWindowsPlatform.h"
+#include "mozilla/StaticPrefs.h"
 #include "mozilla/gfx/DeviceManagerDx.h"
 #include "mozilla/layers/CompositableClient.h"
 #include "mozilla/layers/CompositableForwarder.h"
@@ -256,7 +256,7 @@
   mImageDevice = device;
 
   TextureAllocationFlags allocFlags = TextureAllocationFlags::ALLOC_DEFAULT;
-  if (gfxPrefs::PDMWMFUseSyncTexture() ||
+  if (StaticPrefs::PDMWMFUseSyncTexture() ||
       mDevice == DeviceManagerDx::Get()->GetCompositorDevice()) {
     // If our device is the compositor device, we don't need any synchronization
     // in practice.